lib/screens/cart.dart

```diff
@@ -124,9 +123,9 @@ class _CartPageState extends State<CartPage> {
                                                                 cartItems[index]
                                                                     ['_id'];
                                                             post(
-                                                                Uri.https(
-                                                                    authority,
-                                                                    "users/increaseQuantity"),
+                                                                Uri.parse(
+                                                                    baseUrl +
+                                                                    "/users/increaseQuantity"),
```

Comment:
instead of replacing the code in every file, maybe you can create a helper function ?  for example
getUrl( isDev, baseUrl, path) {
 if (isDev) {
   return Uri.parse(baseUrl + path);
 } else {
   return Uri.https(baseUrl, path);
 }
}

And we can define a constant isDev in a constants file so that it would be very easy for us to switch between the two different implementations.

Reply:
@Uri.parse(baseUrl + path) works for both production and development we need no change every time

Reply:
I see got it